### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by chimwemwe007**
## In this milestone I have achieved the following :

1. Add Users Controller with view placeholder
2. Add Posts Controller with view placeholder
3. setup the routes config

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



You have submitted a project, that uses the outdated README template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/controllers/posts_controllers.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = Post.where(user_id: params[@user.id])
+  end
```
Comment (EricMbouwe) [inline app/controllers/posts_controllers.rb:5]: - [ ] The `user_id` is the one coming from the `@user` that you already have at `line 3`. You would write instead
`Post.where(user_id: @user.id)`

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @chimwemwe007, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Have `users_controller` and `posts_controller` with `#index` and `#show` actions ✔
- Have a corresponding view file for each of the controller actions ✔
- Have users and posts routes added to `config/routes.rb` ✔
- No linter errors ✔

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @chimwemwe007, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**